src/ontology/mondo-ingest.Makefile

```diff
@@ -163,6 +163,23 @@ $(COMPONENTSDIR)/icd10who.owl: $(TMPDIR)/icd10who_relevant_signature.txt | compo
 		remove -T config/properties.txt --select complement --select properties --trim true \
 		annotate --ontology-iri $(URIBASE)/mondo/sources/icd10who.owl --version-iri $(URIBASE)/mondo/sources/$(TODAY)/icd10who.owl -o $@; fi
 
+# TODO: do
+# TODO: remove any unnecessary queries / lines here
+#  - fix_hgnc_mappings
+#  - maybe make a comment about other ones i thought to delete
+$(COMPONENTSDIR)/icd11.owl: $(TMPDIR)/icd11_relevant_signature.txt | component-download-icd11.owl
```

Comment:
## Discussion: Annotation props to remap
@matentzn Trish came up with the good idea of first simply listing out the predicates that we might need to remap.

@twhetzel also further refined my language on this. Is it correct to say that the predicates to map will all be annotation properties? If not, then I'm not capturing everything I might have to remap.

<details><summary>Full list of annotation props</summary>
<p>

browserUrl
codingHint
exclusion
foundationReference
fullySpecifiedName
icd10code
inclusion
isObsolote
longDefinition
narrowerTerm
note
releaseDate
releaseId
http://www.w3.org/2004/02/skos/core#altLabel
http://www.w3.org/2004/02/skos/core#broaderTransitive
http://www.w3.org/2004/02/skos/core#definition
http://www.w3.org/2004/02/skos/core#narrowerTransitive
http://www.w3.org/2004/02/skos/core#prefLabel

</p>
</details> 

<details><summary>Standard remappings already handled</summary>
<p>

We already have `config/property-map-2.sssom.tsv` which will be run. The entries intersection between this and what's in ICD11 is:
- http://www.w3.org/2004/02/skos/core#prefLabel --> http://www.w3.org/2000/01/rdf-schema#label
- http://www.w3.org/2004/02/skos/core#altLabel	--> oboInOwl:hasExactSynonym


</p>
</details> 

## Decisions to be made
<details><summary>Can probably ignore / filter out</summary>
<p>

- releaseDate
- releaseId
- browserUrl
- codingHint

</p>
</details> 

<details><summary>New cases to handle</summary>
<p>

#### `isObsolote` --> `owl:deprecated`
Note that they misspelled 'obsolete'.

#### `note` --> `rdfs:comment`

#### `fullySpecifiedName` --> `oboInOwl:hasExactSynonym`
Open to other suggestions. Note that they're also already using `skos:prefLabel` and `skos:altLabel`, which we will be remapping.

#### Mapping related
Props they have:
- http://www.w3.org/2004/02/skos/core#broaderTransitive
- http://www.w3.org/2004/02/skos/core#narrowerTransitive
- narrowerTerm

I'm not sure if these map to anything outside of ICD11. If they don't, should we simply ignore?
I imagine `narrowerTerm` is either `skos:narrowMatch` or `skos:broadMatch`.

The `skos:broaderTransitive` and `skos:narrowerTransitive` don't have any usages.

#### Definition related
Props they have:
- http://www.w3.org/2004/02/skos/core#definition
- longDefinition

Possible solutions:
a. Combine both and put into IAO:0000115
- Add subheaders?: "short definition" and "long definition"?
b. Put one into IAO:0000115, and include the other using its original predicate, or add it to rdfs:comment

#### Other
`foundationReference`
- include but also put inside rdfs:comment?

`icd10code`
- include but also put inside rdfs:comment?

`inclusion`
- take a look. include but also put inside rdfs:comment?

`exclusion`
- take a look. include but also put inside rdfs:comment?

</p>
</details> 

### Questions
1. In SPARQL, how do I map that which has no URI?
For some props, instead of URIs I'm seeing this `:PRED` syntax, e.g. `Declaration(AnnotationProperty(:isObsolote))` instead of `Declaration(AnnotationProperty(<http://www.w3.org/2004/02/skos/core#altLabel>))`

Reply:
Very good effort to do this properly.

isObsolote --> owl:deprecated --> YES (important)
note --> rdfs:comment --> OK but we will never sync this. May help with debugging though

> The skos:broaderTransitive and skos:narrowerTransitive don't have any usages.

Well then, ditch.

> narrowerTerm

--> ignore for now and make separate issue with examples

> icd10code 

--> hasDbXref: "ICD10WHO:xyz" (needs sparql update)

> fullySpecifiedName --> oboInOwl:hasExactSynonym

Cool, but do in sparql, not remap, and add synonym type "fully qualified names"

> Definition related

Very important, and hard to answer. **I think separate issue**, for now mak skos:definition to IAO:0000115 but in issue, suggest what you want for the longDefinition.

> inclusion/exclusion

Make separate issue with examples. (PRs are not great for this)

> In SPARQL, how do I map that which has no URI?

In the prefixes there should be something like

```
Prefix(:=<http://purl.obolibrary.org/obo/mondo-ingest.owl#>)
```

Which denotes the base iri.

If there is no such thing, I dont know, I guess you will have to try.

Reply:
`icd10code`
Ah, yes I agree it should be `hasDbXref`. I actually read it as 11 and not 10.

_In SPARQL, how do I map that which has no URI?_
Ahh, I see your answer. Excellent; I found: `Prefix(:=<http://id.who.int/icd/schema/>)`

"Will do!" and "Good to hear!" regarding all the other topics!

Reply:
Marking resolved as the only stuff left here will be done in different PR.